.tool-versions

```diff
@@ -1,2 +1,2 @@
-ruby 2.7.6
+ruby 2.7.4
```

Comment:
Ah. Just remove the line from .tool-versions. We already have a .ruby-version file.